src/main/scala/swiss/dasch/domain/AssetFilename.scala

```diff
@@ -13,7 +13,7 @@ final case class AssetFilename private (value: String) extends AnyVal
 
 object AssetFilename {
   // Allow letters, numbers, underscores, hyphens, spaces, full stops, comma, single quote, apostrophe and braces
-  private val regex = """^[\p{L}\p{N}_\- .,'`()]+$""".r
+  private val regex = """^[\p{L}\p{N}_\- .,'`()+=!?<>|]+$""".r
```

Comment:
@seakayone I am currently adding test files with special chars to the dsp-tools tests, and realized that question marks are still not supported, although you have added it here. Your regex looks correct, so I don't know why it doesn't work.

1. Navigate to https://app.dev.dasch.swiss/project/vNYc6-z_ST-a_ENjDxSZjQ/ontology/postcards/Postcard/add
2. Upload this file, named `test?.png`: 

![test?](https://github.com/user-attachments/assets/c80770d2-8d81-40fc-92ec-c13849509b57)

Reply:
Please report it in the app as a bug, it is not urlencoding the filename.